### PR TITLE
Port to tvOS

### PR DESCRIPTION
**Pull request by Michelasso**
I have ported the framework to tvOS. Also added all tests, similarly to macOS, and they all show green. I have tested it a bit in a small app I am making and and least the couple of API calls I made work fine. I am not sure if anything must be done with the Pods, I am not very skilled on that.

Then if you check the diff you'll see that I had to convert some "as!" to "as?". They always gave me a warning no matter the platform. Now after building all is clear.

### CODE REVIEW

File: Sources/Models/ConfigurationMDB.swift [hunk 1]
```diff
@@ -23,12 +23,12 @@ public struct ConfigurationMDB {
...
-    profile_sizes = images["profile_sizes"].arrayObject as! [String]
-    still_sizes = images["still_sizes"].arrayObject as! [String]
-    change_keys = results["change_keys"].arrayObject as! [String]
+    backdrop_sizes = images["backdrop_sizes"].arrayObject as? [String]
+    logo_sizes = images["logo_sizes"].arrayObject as? [String]
+    poster_sizes = images["poster_sizes"].arrayObject as? [String]
+    profile_sizes = images["profile_sizes"].arrayObject as? [String]
+    still_sizes = images["still_sizes"].arrayObject as? [String]
+    change_keys = results["change_keys"].arrayObject as? [String]
   }
```
Comment (gkye) [inline Sources/Models/ConfigurationMDB.swift:32]: thanks for catching these 😓

**Review by gkye — APPROVED** [review]

File: Sources/Models/ConfigurationMDB.swift (hunk 1 above)
Comment (Michelasso) [reply Sources/Models/ConfigurationMDB.swift:32]: Sure, no probs! And thanks for merging!!

Also i was thinking about changing the project to make the frameworks universal, so they can be imported as "import TMDBSwift" in all platforms, basically giving it the value of the TARGET variable (I think. I'll have to double check). The only downside is that all frameworks will have the same name (TMDBSwift.framework)..  I have found a tutorial and that's how they did it as well.

Comment (gkye) [reply Sources/Models/ConfigurationMDB.swift:32]: sounds like a great idea. Im not too sure of the negative side effects of the frameworks sharing the same name. Looking forward to your PR., let me know if you need a hand 😄

Comment (Michelasso) [reply Sources/Models/ConfigurationMDB.swift:32]: Sure thing! In the next few days. 😊

**Pull request merged**